dll/win32/mciavi32/mmoutput.c

```diff
@@ -398,6 +398,12 @@ BOOL MCIAVI_GetInfo(WINE_MCIAVI* wma)
 
 	mmioAscend(wma->hFile, &mmckInfo, 0);
     }
+
+    /* Empty file */
+    if (alb.numVideoFrames == 0) {
+        return FALSE;
```

Comment:
Maybe add a WARN trace?

Reply:
Good idea, same for the other place too (was the idea of the initial patch btw)

Reply:
Done :+1: